src/main/java/rx/internal/operators/OperatorSkipTimed.java

```diff
@@ -77,4 +72,15 @@ public void onCompleted() {
             }
         };
     }
+    
+    final static class AtomicBooleanAction extends AtomicBoolean implements Action0 {
```

Comment:
If you subclass `Subscriber`, implement `Action0`, and use a volatile field for the boolean then you can save even more!